docs/user_guide/trace.md

```diff
@@ -317,3 +426,56 @@ The meaning of the trace timestamps is:
     scheduling.
 
   * BACKEND_OUTPUT: The tensor in the response of a backend.
+
+## OpenTelemetry trace support
+
+Triton provides an option to generate and export traces 
+using [OpenTelemetry APIs and SDKs](https://opentelemetry.io/). 
+
+To specify OpenTelemetry mode for tracing, specify the `--trace-config` 
+flag as follows:
+
+```
+$ tritonserver --trace-config mode=opentelemetry \
+    --trace-config opentelemetry,url=<endpoint> ...
+```
+
+### OpenTelemetry trace APIs settings
+
+The following table shows available OpenTelemetry trace APIs settings for 
+`--trace-config opentelemetry,<setting>=<value>`.
+<table>
+  <thead>
+  <tr>
+    <th>Setting</th>
+    <th>Default Value</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+  <tbody>
+    <tr>
+    <td><code>url</code></td>
+    <td><code>0.0.0.0:4318/v1/traces</code></td>
+    <td>
+      <code>host:port</code> to which the receiver is going to receive 
+      trace data.
```

Comment:
nit: The user isn't supposed to specify the route `/v1/traces/` part, are they? If not, make the default value in the table for `url` `0.0.0.0:4318` and note in the description that `/v1/traces` will be called at this `url`

Reply:
I think it would be better to report default value the same as OpenTelemetry has for HTTP exporter:
https://github.com/open-telemetry/opentelemetry-cpp/blob/v1.8.3/exporters/otlp/src/otlp_environment.cc#L139

I can add clarification that `/v1/traces` is not strongly required, for example?

Reply:
Also link to OTLP README, which I would like to follow for the default values: https://github.com/open-telemetry/opentelemetry-cpp/tree/v1.8.3/exporters/otlp#configuration-options--otlp-http-exporter-

Let me know what you think.

Reply:
`/v1/traces` won't be added to provided url - it wasn't added in my test, so I just collected on some port without `/v1/traces`. So if `url` is provided, it will be used as is. `/v1/traces` is added if environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` is set: https://github.com/open-telemetry/opentelemetry-cpp/blob/v1.8.3/exporters/otlp/src/otlp_environment.cc#L153